docker-compose.yml

```diff
@@ -256,7 +256,8 @@ services:
       KEYCLOAK_ADMIN_PASSWORD: ${KEYCLOAK_ADMIN_PASSWORD}
       WEB_API_BASE_URL: ${FORMSFLOW_API_URL}
       EMPLOYEE_DATA_API_URL: ${EMPLOYEE_DATA_API_URL}
-      EMPLOYEE_DATA_AUTH_TOKEN: ${EMPLOYEE_DATA_AUTH_TOKEN}
+      ODS_AUTH_TOKEN: ${ODS_AUTH_TOKEN}
```

Comment:
The `EMPLOYEE_DATA_AUTH_TOKEN` is also used in the Openshift templates for camunda and the API, so you'll have to update those as well, or revert the change

Reply:
I thought `ODS_AUTH_TOKEN` is more general, now that we use it for more than one endpoint. Will update the Openshift once deployed

Reply:
For sure! The templates are defined in the `deployment/openshift` folder and are automatically applied when we merge things, so you'll have to update `camunda_dc.yaml` and `webapi_dc.yaml` before we can merge this 🙃